cumulus_library/studies/core/medication_request.sql

```diff
@@ -1,15 +1,41 @@
 -- ############################################################
 -- FHIR MedicationRequest : Med Orders *possibly* Administered
-CREATE TABLE core__medication_request AS
+CREATE TABLE core__medicationrequest AS
 SELECT
     mr.status,
     mr.intent,
-    mr.authoredon,
+    date(from_iso8601_timestamp(mr.authoredon)) AS authoredon,
+    date_trunc('month', date(from_iso8601_timestamp(mr.authoredon)))
+    AS authoredon_month,
     mr.category,
     med_coding.code AS code,
     med_coding.display AS display,
     med_coding.system AS system, --noqa: RF04
     mr.id AS med_admin_id,
     mr.subject.reference AS subject_id
 FROM medicationrequest AS mr,
-    UNNEST(mr.medicationcodeableconcept.coding) AS t (med_coding); --noqa: AL05
+    unnest(mr.medicationcodeableconcept.coding) AS t (med_coding) --noqa: AL05
+WHERE med_coding.system = 'http://www.nlm.nih.gov/research/umls/rxnorm';
```

Comment:
tsk, no https?  :smile:

Reply:
it does at least redirect to an https url if you try to put it in a browser

Reply:
it does at least redirect to an https url if you try to put it in a browser